service/app/poller.go

```diff
@@ -92,9 +98,12 @@ func sentTransactionsPoller(app *App) {
 	for {
 		select {
 		case <-ticker.C:
+			start := time.Now()
 			log.Trace("SentTransactionsPoller poll start")
```

Comment:
do we wanna make this` log.Trace("SentTransactionsPoller poll start")` to the `info` level as well?

Reply:
Ok, will add it for now, we can remove it later if it gets too verbose